_summaries/wu2018group_2.md

```diff
@@ -1,19 +1,51 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{wu2018}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: # SShivang
+score: # 8/10
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+Core Idea:
+
+The batch norm introduced an optimization in which the outputs of the layers were normalized for a particular batch (scaled and recentered). This lead to significant improvement in performance for a lot of deep learning tasks (one of the most important improvements). However with smaller batch sizes, this technique doesn't work as well. This may be the case if it is impossible to have large batch sizes (models such as video classification models) and you have to train on a smaller batch size. This paper introduces a group norm which works on smaller batches and has consistent improvements of performance across batch sizes. Group norm breaks down does normalization of the input layer by breaking channels into groups.
+
+Technical Realization:
+
+![Performance of Batch Norm vs Group Norm](wu2018group_2_1.png)
+
+Batch norms are highly dependent on batches. However, for tasks such as image segmentation, object detection, and video classification, models train on smaller batch sizes as their inputs are so large. In this case batch norm is less useful.
```

Comment:
I think this repeats the core idea. You could remove it

Reply:
Moved it to earlier section